sfcasts/cor-sf-config-and-nullobject.md

```diff
@@ -53,7 +61,7 @@ So let's get to it! Back in our code, find that `if` we've been talking about,
 which is inside any handler. What we need to do is remove the `if` and call the
 next handler *directly*. *Easy peasy*. Create a new handler class and call
 it `NullHandler`. Make it implement the `XpBonusHandlerInterface` and hold 
-"Option" + "Enter" to implement the methods. And now... *nothing*!
```

Comment:
I just wanted to emphasize that we'll do "nothing" now. That's the purpose of the Null Object pattern, do as close as nothing in their methods